reliability-engineering/terraform/modules/concourse-secrets-admin/iam.tf

```diff
@@ -0,0 +1,87 @@
+resource "aws_iam_role" "concourse_secrets_admin" {
+  name = "${var.deployment}-${var.concourse_team_name}-concourse-secrets-admin"
+
+  assume_role_policy = <<-EOF
+  {
+    "Version": "2012-10-17",
+    "Statement": [
+      {
+        "Effect": "Allow",
+        "Principal": {
+          "Federated": "${module.concourse_base.aws_iam_openid_connect_provider_arn}"
+        },
+        "Action": "sts:AssumeRoleWithWebIdentity",
+        "Condition": {
+          "StringEquals": {
+            "${var.oidc_host_path}:aud": "${var.github_oauth_client_id}",
+            "aws:RequestTag/t${var.trusted_github_team_id}": "t"      // instantiated in team.tf files in tech-ops-private
```

Comment:
Im not sure you can put inline comments within json.

Reply:
eek no you can't well caught (perhaps aws is just lenient here), this is why I think it's preferential to list these things as terraform and use `jsonencode` to turn them into json. then you can still put comments in and do other fun things which will get squashed out when it gets boiled down to json.

Reply:
Pushed conversion to terraform syntax